mobile-app/cypress/integration/functional/wallet/loans/vault_detail.spec.ts

```diff
@@ -0,0 +1,137 @@
+context('Wallet - Loans - Vault Details', () => {
+  let vaultId = ''
+
+  before(function () {
+    cy.createEmptyWallet(true)
+    cy.sendDFItoWallet().sendDFITokentoWallet().sendTokenToWallet(['BTC']).wait(6000)
+    cy.getByTestID('bottom_tab_loans').click()
+    cy.getByTestID('empty_vault').should('exist')
+    cy.createVault(0)
+    cy.getByTestID('vault_card_0_manage_loans_button').should('not.exist')
+    cy.getByTestID('vault_card_0_vault_id').then(($txt: any) => {
+      vaultId = $txt[0].textContent
+    })
+  })
+
+  it('should check empty state', function () {
+    cy.getByTestID('vault_card_0').click()
+    cy.getByTestID('collateral_tab_COLLATERALS').click()
+    cy.getByTestID('button_add_collateral').should('exist')
+    cy.getByTestID('collateral_tab_LOANS').should('have.attr', 'aria-disabled')
+    cy.go('back')
+    cy.wait(2000)
+    cy.getByTestID('vault_card_0_edit_collaterals_button').click()
```

Comment:
Since we are targeting vault details screen, why not we trigger add collateral using `button_add_collateral`?
Also because `vault_card_0_edit_collaterals_button` is already tested in `collateral.spec.ts`

Reply:
Oh right, i mislook that. Will update